File: Crawler Log Folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int count = 0;
        for (string& log: logs) {
            if (log == "../") {
                if (--count < 0) count = 0;
            } else if (log != "./") {
                count++;
            }
        }
        return count;
    }
};

File: Crawler Log Folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        count = 0
        for log in logs:
            if log == "../":
                count -= 1
                if count < 0:
                    count = 0
            elif log != "./":
                count += 1
        return count
